### ISSUE TITLE
how to install this plugin?

### ISSUE THREAD

**Issue by luutuankiet** [roygbyte/sudoku.koplugin#1]
Hi, I'm interested in this plugin, tried your other repo crosswords and it works great. Is there a guide somewhere I can contribute to this ?

Thanks!

**Comment by roygbyte** [comment]
Hi! How would you like to contribute? This project isn't finished yet, but if you need help setting up your environment to make contributions I can try to give you a hand.

**Comment by luutuankiet** [comment]
yes please can you point me to the environment setup ? I don't have experience with lua but can refer to your other repo https://github.com/roygbyte/crossword.koplugin and koreader docs to fix something up.

**Comment by roygbyte** [comment]
Have a look at this article: https://gopher.floodgap.com/gopher/gw?a=gopher%3A%2F%2Froygbyte.com%2F1%2Fhow_i_develop_plugins_for_koreader.gph

It's the process I followed when I was doing this development.

**Comment by archbtww** [comment]
@roygbyte could you help me set up an environment for this?
